packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -142,6 +145,36 @@ export class Ec2Service extends BaseService implements IEc2Service, elb.ILoadBal
     }
   }
 
+  /**
+   * Adds AWS X-Ray daemon as a sidecar container to enable tracing
```

Comment:
Again, describe what the new behavior would be and not how it's going to be implemented. You can add implementation details in the second paragraph

Reply:
Addressed